fv/test-workload/test-workload.go

```diff
@@ -118,20 +128,25 @@ func main() {
 			}
 		}
 
-		l, err := net.Listen("tcp", ipAddress+":"+port)
-		if err != nil {
-			return err
+		// Listen on each port.
+		for _, port := range ports {
+			log.WithField("port", port).Info("About to listen for connections")
+			l, err := net.Listen("tcp", ipAddress+":"+port)
+			panicIfError(err)
```

Comment:
Within the namespace.Do func, I was returning the err, rather than panicking directly, because I wasn't sure that it would be OK to panic while in the workload namespace.  Is that a valid concern at all?

Reply:
Looks like all their cleanup code is nicely `defer`-ed so it just works.

Reply:
OK, cool.